source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/users/list/add/index.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import { Modal } from 'semantic-ui-react';
+import CloseIcon from '@material-ui/icons/Close';
+import AddIcon from '@material-ui/icons/Add';
+import { withTemplateHelpers, Toggle } from 'react-action-decorators';
+import { defaultOptions } from '@data';
+import { getCurrentOrganizationId } from "@lib/current-organization";
+import { i18nProps } from '@lib/i18n';
+import { compose } from 'recompose';
+import { withData, WithDataProps } from 'react-orbitjs';
+import { withTranslations } from '@lib/i18n';
+import UserInput from './user-input';
+
+interface IOwnProps{
+  onUserAdded: () => Promise<void>;
+}
+
+export type IProps = IOwnProps
```

Comment:
do you need to export this type?

Reply:
it doesn't do any harm to export it, as types aren't included in the transpiled code